README.md

```diff
@@ -141,17 +141,17 @@ supporting services for the platform.
 
 ### Deploy
 
-Run the `create-microbosh` pipeline.
+Run the `create-bosh-cloudfoundry` pipeline (which will also deploy CloudFoundry - see below).
```

Comment:
I think we can merge CF and BOSH subsections now as we've got just one pipeline.